opentelemetry-otlp/src/exporter/http/mod.rs

```diff
@@ -291,6 +312,26 @@ fn resolve_endpoint(
         .map_err(From::from)
 }
 
+fn add_header_from_string(input: &str, headers: &mut HashMap<HeaderName, HeaderValue>) {
+    for pair in input.split(',') {
+        if pair.is_empty() {
+            continue;
+        }
+        let mut kv_iter = pair.splitn(2, '=');
+        match (kv_iter.next(), kv_iter.next()) {
```

Comment:
nit: matching on [split_once](https://doc.rust-lang.org/stable/std/primitive.str.html#method.split_once) is simpler

Reply:
Good point. Done.